Jayme/Logger.swift

```diff
@@ -35,7 +35,7 @@ open class Logger {
     // MARK: - Private
     
     internal var requestCounter = 0
-    internal func log(_ items: Any..., _ separator: String = " ", _ terminator: String = "\n") {
+    internal func log(_ items: Any..., separator: String = " ", _ terminator: String = "\n") {
```

Comment:
This one is interesting, it doesn't let seem to want me to have more than one unlabeled parameter with a default. I did not design the API so I'm not 100% sure what's more intuitive, so I had to pick one.

Reply:
By using common sense, I'd say that the first one is OK to be unlabeled, whereas the rest do need the label to know what they are about. The function should end up like this:
```
internal func log(_ items: Any..., separator: String = " ", terminator: String = "\n")
```

Notice that `terminator` is also labeled. The function can be called like this:

```
log("hello", "world", "abc", "123", separator: " ", terminator: "/n")
```

I guess the reason why it won't let you use a unlabeled second parameter it's because the first one is a variadic one (i. e. you can send as many items as you want separated by commas, so, there must be a way to indicate when the list has ended and the second parameter begins).